package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "workerpool",
-  "version": "1.0.0",
```

Comment:
Why are we downgrading the version number?

Reply:
I was upgrading it to the current version, idk thought it'd be nice. But is this not what I should do?

Reply:
OH. I misread the diff somehow. Facepalm sorry!!